### ISSUE TITLE
Disable dismissal until all permissions have been allowed

### ISSUE THREAD

**Issue by feelingsonice** [jevonmao/PermissionsSwiftUI#160]
Right now when the user denies a permission it shows a "DENIED" button and tapping on it does nothing. Furthermore, the user is able to dismiss the alert once they've interacted with every permission, even if some permissions have been denied.

I'd look for a way to do the following:

1. If the user denied a permission, instead of a "DENIED" button that does nothing, open the settings page. This can be done via `UIApplication.shared.open(URL(string: UIApplication.openSettingsURLString)!)`
2. A way to disable dismissal of the alert entirely until all permission have been allowed
3. Have the auto check authorization feature respect 2). In other words, there's no "X" button on the top right anymore, the view is dismissed automatically once the user allowed every single permission requested.

Basically, I'm looking for a way to force the user to allow every permission requested in order to use the app. And provide the user with a button that takes them to settings if they denied a permission. Is there a way to achieve this right now?
